src/brackets.js

```diff
@@ -205,6 +206,7 @@ define(function (require, exports, module) {
             ScrollTrackMarkers      : require("search/ScrollTrackMarkers"),
             UpdateNotification      : require("utils/UpdateNotification"),
             WorkingSetView          : WorkingSetView,
+            XMLUtils                : XMLUtils,
```

Comment:
This isn't normally needed -- only if you're going to do popup-window unit tests (basically integration tests) that need to talk to this module, and you don't have any unit tests that are set up that way (nor should you, for a non-UI module like this).  Fyi @MarcelGerber